src/scanner/custom-rules/color-rule.ts

```diff
@@ -13,7 +13,7 @@ export const colorConfiguration: RuleConfiguration = {
     ],
     rule: {
         id: 'select-body',
-        selector: 'body',
+        selector: '*',
```

Comment:
I think the `select-body` id is misleading now that this is selecting more than just the body element. Could you update the ID to something that reflects the new selector?

Reply:
Done
I renamed this and other related ids in my last commit.

Reply:
Thanks! However, in this context, the term "document" has a specific meaning which is still different from what's being selected here. Maybe `select-each-element` or `select-all-elements`?

Reply:
I would also extract the name out to a variable so you can reference that in `use-of-color.tsx` and `visualization.tsx`.

Reply:
Makes sense to me
Is there a specific file in the project where we put those common variables @waabid ?

Reply:
I think the `color-rule.tsx` file should be fine for this purpose in this case.